lib/service/services_cli.rb

```diff
@@ -50,7 +50,7 @@ def user_of_process(pid)
       if pid.nil? || pid.zero?
         user
       else
-        Utils.safe_popen_read("ps", "-o", "user", "-p", pid.to_s).lines.second.chomp
+        Utils.safe_popen_read("ps", "-o", "user", "-p", pid.to_s).lines.second&.chomp
```

Comment:
This will return `nil` which seems undesirable. How about:
`Utils.safe_popen_read("ps", "-o", "user", "-p", pid.to_s).lines.second.to_s.chomp`